docs/releases/4.0.txt

```diff
@@ -226,7 +226,8 @@ Internationalization
 Logging
 ~~~~~~~
 
-* ...
+* The alias of the database used in a SQL call is now passed as extra context
```

Comment:
```suggestion
* The alias of the database used in an SQL call is now passed as extra context
```

Reply:
Ok. I went with what you have based on that being using in the [Wikipedia article](https://en.wikipedia.org/wiki/SQL).

It's actually complicated based on how you pronounce it though:

1. "S-Q-L" → ...an SQL...
2. "Sequel" → ...a SQL...

Fun.

Reply:
I know, but in Django's docs we prefer `an`.

Reply:
Now I know!